homeassistant/components/media_player/yamaha.py

```diff
@@ -0,0 +1,126 @@
+"""
+Support for Yamaha Receivers.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/media_player.yamaha/
+"""
+import logging
+import rxv
+
+from homeassistant.components.media_player import (
+    DOMAIN, SUPPORT_NEXT_TRACK, SUPPORT_PAUSE, SUPPORT_PREVIOUS_TRACK,
+    SUPPORT_TURN_OFF, SUPPORT_TURN_ON, SUPPORT_VOLUME_MUTE, SUPPORT_VOLUME_SET,
+    MediaPlayerDevice)
+from homeassistant.const import CONF_HOST, STATE_OFF, STATE_ON, STATE_UNKNOWN
+
+_LOGGER = logging.getLogger(__name__)
+_GLOBALRECEIVER = None
+
+SUPPORT_YAMAHA = SUPPORT_VOLUME_SET | SUPPORT_VOLUME_MUTE | \
+    SUPPORT_TURN_ON | SUPPORT_TURN_OFF
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Setup the Denon platform."""
+    yamaha = YamahaDevice()
+    if yamaha.update():
+        add_devices([yamaha])
+        return True
+    else:
+        return False
+
+
+class YamahaDevice(MediaPlayerDevice):
+    """Representation of a Yamaha device."""
+
+    # pylint: disable=too-many-public-methods, abstract-method
+    def __init__(self):
+        self._name = "Yamaha Receiver"
+        self._host = ""
+        self._pwstate = "Off"
+        self._muted = False
+        self._mediasource = ""
+
+    def update(self):
+        """Get the latest details from the device."""
+        try:
+            receivers = rxv.find()
+            global _GLOBALRECEIVER
+            _GLOBALRECEIVER= receivers[0]
+        except ConnectionRefusedError:
+            return False
+        
+        if _GLOBALRECEIVER.on == True:
```

Comment:
Instead of testing against `True` and `False` you can just do this:
```python
if _GLOBALRECEIVER.on:
    self._pwstate = STATE_ON
else:
    self._pwstate = STATE_OFF
```